programmers/p-y.js

```diff
@@ -0,0 +1,17 @@
+function solution(s){
+    var leng = s.length
+    if (leng<=50){
+        var num1 = s.match(/p/ig);
+        var num2 = s.match(/y/ig);
+        if (num1.length == num2.length){
+            return true;
+        }
+        else{
+            return false;
+        }
+    }
+    else {
+        console.log("다시 실행하세요.");
+    }
+}    
```

Comment:
정규표현식도 쓰셨네요!! 저보다 잘하시는거같아요..
if문 안에서 return true가 있어서 else문은 필요없습니다.

```suggestion
    if (linearContent <= 50) {
        let num1 = s.match(/p/ig);
        let num2 = s.match(/y/ig);
      
        if (num1.length === num2.length) {
            return true;
        } 
    }
}    
```